### PR TITLE
Support for relative width

### PR DESCRIPTION
**Pull request by AnonymouX47**
- The `width` option can now accept an integer percentage as its value.
- When `width` is relative, `height` is ignored and a warning is logged.
- Updates the descriptions of `height` and `width` options and uses relative width in the existing example.
- Corrects the options used in the example code block to match what is actually used in the video directive.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -50,27 +50,30 @@ the video directive supports all the optional attributes from the html tag as su
...
 They can be used as any directive option:
 
 .. code-block:: rst
 
     .. video:: _static/video.mp4
         :nocontrols:
+        :autoplay:
+        :muted:
         :loop:
-        :poster: _static/image.png
```
Comment (12rambau) [inline docs/quickstart.rst:67]: why did you removed the poster image ?

File: docs/quickstart.rst [hunk 2]
```diff
@@ -50,27 +50,30 @@ the video directive supports all the optional attributes from the html tag as su
...
+        :width: 100%
```
Comment (12rambau) [inline docs/quickstart.rst:69]: does it mean that 100% is not default anymore ?

**Review by 12rambau — COMMENTED** [review]
what do you think if we go one step further and accept any valid css descriptor ? (em, rem, %, vw, vh, pixel)
If you don't have time it can be done in a follow-up PR.

File: docs/quickstart.rst (hunk 2 above)
Comment (AnonymouX47) [reply docs/quickstart.rst:69]: No, it isn't. The default (i.e without `width` or `height`) is to use the video's original resolution.

For instance, check the Quickstart page of the `latest` version of docs and compare it to the version of the docs built from this PR.

File: docs/quickstart.rst (hunk 1 above)
Comment (AnonymouX47) [reply docs/quickstart.rst:67]: It was in the code block but not in the actual `videp` directive.

**Comment by AnonymouX47** [comment]
Actually, have little to no knowledge of HTML and CSS.

I just searched up how to do this specifically and mokey-patched the `Video` class from within my `conf.py`. Then, I thought it could actually be useful as a feature and decided to contribute it.

**Comment by Borda** [comment]
This would be great to land together with `max-width` :rabbit:

File: docs/quickstart.rst (hunk 1 above)
Comment (12rambau) [reply docs/quickstart.rst:67]: it was at least so let's keep it for now and open a dedicated issue if it's never used.

**Review by 12rambau — CHANGES_REQUESTED** [review]
@AnonymouX47 sorry for the long wait, all my OSS time was eaten up by another project. I would like to move forward with your PR, could you rebase it with the current main so I can merge it in the next release ?

**Comment by AnonymouX47** [comment]
All done, I believe.

**Review by 12rambau — APPROVED** [review]

**Pull request merged**